src/client.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import matplotlib.pyplot as plt
+import sys
+from Errors import WrongSettlementException
+
+
+
+def display_disribution(settlement:str, host:str, port:str):
+    param = {"settlement":settlement}
+    url = f"http://{host}:{port}/get_distribution"
+    
+    response = requests.get(url, params= param)
+    print(response.status_code)
+    if response.status_code == 200:
+        # in case the settlement is exist but no alarms were there
+        if "message" in response.json():
+            sentence = response.json().get('message', '')
+            print(sentence)
+        response_lst = response.json()
+        hour_df = [d['hour'] for d in response_lst]
```

Comment:
its not df (dataframe) its a vector

```suggestion
        hours_vector = [d['hour'] for d in response_lst]
```